src/chttpd/src/chttpd.erl

```diff
@@ -326,7 +326,7 @@ handle_request_int(MochiReq) ->
          || Part <- string:tokens(Path, "/")
         ],
         requested_path_parts = [
-            ?l2b(unquote(Part))
+            ?l2b(chttpd:unquote(Part))
```

Comment:
We can keep using the local call here I think?

Reply:
I have changed it back to calling the local one and the same for the call above it.